app/course/controllers/CourseCtrl.js

```diff
@@ -149,11 +149,15 @@ courseApp.controller('CourseCtrl', ['$scope', '$rootScope', '$routeParams', '$ti
 			}
 		};
 
-		$scope.submitAssignTagTooltip = function() {
-			courseActionCreators.submitAssignTagTooltip(
+		$scope.submitMassAssignTags = function() {
+			courseActionCreators.submitMassAssignTags(
 				$scope.view.tagOccurences,
```

Comment:
Your PR shouldn't involve touching this file, I suspect something went wrong git-ing, I'll look at it with you next time we're in the office.